newdle/client/src/selectors.js

```diff
@@ -104,10 +104,14 @@ export const getParticipantAvailability = createSelector(
   getNewdleParticipants,
   (timeslots, participants) => {
     return timeslots.map(timeslot => {
+      console.log('time', timeslot);
       const available = participants
-        .filter(part => ['available', 'ifneedbe'].includes(part.answers[timeslot]))
+        .filter(part => ['available'].includes(part.answers[timeslot]))
         .map(part => ({...part, fullyAvailable: part.answers[timeslot] === 'available'}));
-      return {startDt: timeslot, available};
+      const ifneedbe = participants
```

Comment:
This is pretty much repeating the statement above. I wonder if it wouldn't be nicer to define an object before (with available and ifneedbe set to 0), and then just using `.forEach()` and incrementing the respective counter in that object.

Reply:
I think I followed you. I tried to implement `.forEach() in the most recent push`

Reply:
Thanks, I'll have a closer look at the updated selector tomorrow... need to double-check what data we have there before commenting ;)

Reply:
No problem - thanks for bearing with me on this commit.